modmail/utils/cogs.py

```diff
@@ -0,0 +1,40 @@
+from dataclasses import dataclass
+from enum import IntEnum
+
+
+class BotModes(IntEnum):
+    """
+    Valid modes for the bot.
+
+    These values affect logging levels, which logs are loaded, and so forth.
+    """
+
+    production = int("1", 2)
+    develop = int("10", 2)
+    plugin_dev = int("100", 2)
```

Comment:
1>>0
1>>1
1>>2 
is neater

Reply:
Actually:
```py
class BitwiseAutoEnum(IntEnum):
  def _generate_next_value_(name, start, count, last_values):
    return 1>>count
```

Reply:
How does that work?

Reply:
@onerandomusername aren't these just logical shifts?